.github/workflows/k8s-patch-os-jobs.yml

```diff
@@ -31,6 +35,10 @@ jobs:
   create-k8s-on-ec2:
     runs-on: ubuntu-latest
     steps:
+      # Sleep 20 seconds between each job's start
+      - name: Sleep to avoid throttling limits
+        run: sleep $((${{ env.JITTER_INDEX }} * 20))
```

Comment:
Maybe you could also utilize the {RANDOM} function to randomize the jitter a bit too. Not a big deal though

Reply:
Thought of that, but better to have it controlled and run in a predictable sequential manner than to randomize and potentially run into the same issue again if we get unlucky IMO.

Good callout though